test/unit/accord/utils/random/Picker.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package accord.utils.random;
```

Comment:
The whole general `accord.utils` package naming is something I suppose we should address in the long term if the thing in it are really not Accord-specific. No sense in changing before `cep-15-accord` merges though, of course.

Reply:
i think there are a few things...

1) we have several testing utils that *should* have a different life cycle than accord, so likely should be moved out for better isolation
2) when cep-15-accord rebases this forward part has caused issues... the cep-15-accord history hasn't been squashed so all changes to these packages exist in cep-15-accord history, causing conflicts on rebase to trunk...  to avoid this i added a commit on top of trunk last time i did the rebase to rename this package; i then removed after the rebase completed...